README.md

```diff
@@ -61,6 +61,41 @@ return <ConsolePage title="App Title" routes={routes} />;
 > [!TIP]
 > When you create Falcon navigation links for this multi-page application, make sure the `navigation.links[].path` in `manifest.yml` match the `path` provided in the `routes` attribute.
 
+### `CollectionEditor`
+
+Component that allows collection objects to be read, modified, and deleted as JSON. You can determine the extent to which an end user can modify collection name and object name to interact with, as well as which actions are available to the user.
+
+Using the CollectionEditor doesn't _create_ collections. To do that, you need to define the collection in `manifest.yml`. See the Foundry docs.
+
+**Basic collection editor**
```

Comment:
Should we put some note that basically says that you need to know the collection and object name to actually use this component? At least until we get the list APIs.